gravitee-apim-console-webui/src/management/home/home-overview/home-overview.component.html

```diff
@@ -56,27 +56,25 @@
   <h2 class="home-overview__row-title">V2 APIs</h2>
   <div class="home-overview__row-cards">
     <mat-card class="home-overview__card">
-      <mat-card-content>
-        <div class="mat-body-1">API Response Status</div>
-
+      <div class="home-overview__card__header">API Response Status</div>
+      <mat-card-content class="home-overview__card__content">
         <ng-container *ngIf="apiResponseStatus; else loader">
           <gio-api-response-status [data]="apiResponseStatus"></gio-api-response-status>
         </ng-container>
       </mat-card-content>
     </mat-card>
     <mat-card class="home-overview__card">
-      <mat-card-content>
-        <div class="mat-body-1">
-          Top APIs
-          <span class="mat-caption">(Ordered by API calls)</span>
-        </div>
-      </mat-card-content>
+      <div class="home-overview__card__header">
+        Top APIs
+        <mat-icon class="home-overview__card__header__tooltip" svgIcon="gio:info" matTooltip="Ordered by API calls"></mat-icon>
```

Comment:
A bit too much ;-). class name should not be that long, too verbose , we know we are in home-overview already :-) 
2 levels is enough, then reset, sometimes you add so called --modifier:
```
.person--handsome { }
.person__face--handsome { }
```

docs: https://cssguidelin.es/#bem-like-naming

Reply:
👍 fixed